build.gradle

```diff
@@ -62,13 +66,26 @@ subprojects {
 
     dependencies {
         compile "org.jetbrains.kotlin:kotlin-reflect"
-        compile "junit:junit:$junit_version"
         compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
         compile "org.jetbrains.kotlin:kotlin-test:$kotlin_version"
         compile "com.jayway.restassured:rest-assured:$rest_assured_version"
         compile "org.slf4j:slf4j-api:$org_slf4j_version"
         compile "io.github.jupf.staticlog:staticlog:$static_log_version"
         compile "com.google.guava:guava:$guava_version"
+        compile "io.kotlintest:kotlintest-runner-junit5:$kotlin_test_version"
+        compile "org.junit.platform:junit-platform-suite-api:$junit_platform_version"
+        compile "org.junit.platform:junit-platform-runner:$junit_platform_version"
+
+        testImplementation(
+                "org.junit.jupiter:junit-jupiter-api:$junit_jupiter_version"
+        )
+        testRuntimeOnly(
+                "org.junit.jupiter:junit-jupiter-engine:$junit_jupiter_version",
+        )
```

Comment:
✏️ The gradle release notes on JUnit5 support has it like
`testImplementation 'org.junit.jupiter:junit-jupiter-api:5.1.0'` (instead of having the parentheses).
✏️ Also you have an unneeded comma at the end of line 83.

Reply:
Will check and will fix.